index.html

```diff
@@ -60,7 +60,156 @@ <h2 id="i-am-a">I’m a software developer</h2>
+					</section>
+					<!-- Card -->
+					<section class="recent-works__card">
+						<div class="recent-works__img">
+							<img src="" alt="" >
+							<div class="recent-works__info">
+								<h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+								<ul class="recent-works__tags">
+									<li class="recent-works__tag">Ruby on rails</li>
+									<li class="recent-works__tag">css</li>
+									<li class="recent-works__tag">JavaScript</li>
+									<li class="recent-works__tag">html</li>
+								</ul>
+								<a class="recent-works__link" href="#">See Project</a>
+							</div>
+						</div>
+					</section>
+					<!-- Card -->
+					<section class="recent-works__card">
+						<div class="recent-works__img">
+							<img src="" alt="" >
+							<div class="recent-works__info">
+								<h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+								<ul class="recent-works__tags">
+									<li class="recent-works__tag">Ruby on rails</li>
+									<li class="recent-works__tag">css</li>
+									<li class="recent-works__tag">JavaScript</li>
+									<li class="recent-works__tag">html</li>
+								</ul>
+								<a class="recent-works__link" href="#">See Project</a>
+							</div>
+						</div>
+					</section>
+					<!-- Card -->
+					<section class="recent-works__card">
+						<div class="recent-works__img">
+							<img src="" alt="" >
+							<div class="recent-works__info">
+								<h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+								<ul class="recent-works__tags">
+									<li class="recent-works__tag">Ruby on rails</li>
+									<li class="recent-works__tag">css</li>
+									<li class="recent-works__tag">JavaScript</li>
+									<li class="recent-works__tag">html</li>
+								</ul>
+								<a class="recent-works__link" href="#">See Project</a>
+							</div>
+						</div>
+					</section>
+					<!-- Card -->
+					<section class="recent-works__card">
+						<div class="recent-works__img">
+							<img src="" alt="" >
+							<div class="recent-works__info">
+								<h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+								<ul class="recent-works__tags">
+									<li class="recent-works__tag">Ruby on rails</li>
+									<li class="recent-works__tag">css</li>
+									<li class="recent-works__tag">JavaScript</li>
+									<li class="recent-works__tag">html</li>
+								</ul>
+								<a class="recent-works__link" href="#">See Project</a>
+							</div>
+						</div>
+					</section>
+				</section>
+			</section>	
+			
+		</section>
+		<section id="about-me">
+			<div class="about-me__text">
+				<h2 class="about-me__primary-text">About me</h2>
+				<p class="about-me__supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+				<a class="about-me__link" href="#">Get My Resume</a>
+			  </div>
```

Comment:
- [ ] Good job on the project 👏 . For this section, kindly fit in the background image just as it is on the figma design

## Figma:
![image](https://user-images.githubusercontent.com/50990925/219907460-db905a98-c603-4872-9427-69d854127f11.png)

## Your design:
![image](https://user-images.githubusercontent.com/50990925/219907470-7c40bbef-eefa-4cb5-9b9e-71da5052e8f2.png)